.NET/Microsoft.Recognizers.Text.DateTime/Japanese/Extractors/DurationExtractorJap.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+using Microsoft.Recognizers.Definitions.Japanese;
+using Microsoft.Recognizers.Text.Number;
+using Microsoft.Recognizers.Text.NumberWithUnit.Japanese;
+using Microsoft.Recognizers.Text.NumberWithUnit;
+
+using DateObject = System.DateTime;
+
+namespace Microsoft.Recognizers.Text.DateTime.Japanese
+{
+    public class DurationExtractorJap : BaseDateTimeExtractor<DurationType>

```

Comment:
If most of the logic of extractor/parser classes are shared between Chinese and Japanese, it would be best to call them in a pattern like CJKDurationExtractor and put them in the Microsoft.Recognizers.Text.DateTime.Extractor and Microsoft.Recognizers.Text.DateTime.Parser
But I'm fine with merging this way, and refactoring to unify later.